### PR TITLE
Add UKPRN to Rider acronym list

### PR DESCRIPTION
**Pull request by timabell**
To stop Jetbrains' Rider complaining about the spelling of UKPRN (United Kingdom Provider Reference Number)

### CODE REVIEW

File: CourseDirectory.All.sln.DotSettings [hunk 1]
```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=ukprn/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```
Comment (gunndabad) [inline CourseDirectory.All.sln.DotSettings:2]: Nit: indentation is whacky here

Comment (timabell) [reply CourseDirectory.All.sln.DotSettings:2]: This is how Rider generates it. I'm not sure it's a file I'd generally look in. I'm guessing if we fix the indentation Rider would just blat it again.

**Review by gunndabad — APPROVED** [review]

File: CourseDirectory.All.sln.DotSettings (hunk 1 above)
Comment (timabell) [reply CourseDirectory.All.sln.DotSettings:2]: It's a shame they haven't got a nice cross platform variant. I've raised a ticket https://rider-support.jetbrains.com/hc/en-us/requests/3028029

**Comment by timabell** [comment]
Branch for shared version that doesn't actually help with Rider https://github.com/SkillsFundingAgency/dfc-coursedirectory/compare/add-code-analysis-dictionary

**Comment by timabell** [comment]
https://youtrack.jetbrains.com/issue/RSRP-218768 Rider feature request

**Pull request merged**